src/itemPublished/itemPublished.ts

```diff
@@ -23,6 +23,18 @@ export interface PackedItemPublished {
   totalViews: number;
 }
 
+export enum PublicationStatus {
+  Unpublished = 'unpublished',
+  Published = 'published',
+  PublishedChildren = 'publishedChildren',
```

Comment:
`PublishedChildren` means the current item is published because it is a child of an already published folder. I don't know if it is more understandable and if I should change this enum key ?